dlk/hw/make_os/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y gcc-8-aarch64-linux-gnu \
+make \
+sudo \
+gcc \
+bison \
+flex \
+device-tree-compiler \
+debootstrap \
+qemu-user-static \
+vim \
+&& rm -rf /var/lib/apt/lists/*
```

Comment:
Why do you need gcc (for aarch64), make, sudo, ...etc?
It seems debootstrap and qemu-user-static are needed, but others aren't needed in later script...

Reply:
@primenumber 
The tools for kernel compilation (This is later work, not in this PR) is included also. Sorry for confusing code.
I'll comment out and add comment to the redundant part.